src/internal/events.ts

```diff
@@ -1,5 +1,5 @@
 import { JsonStruct } from '@metamask/utils';
-import { literal, object } from 'superstruct';
```

Comment:
We should use `exactOptional` here

Reply:
Addressed here [c125539](https://github.com/MetaMask/keyring-api/pull/307/commits/c12553984bde91b656adbdc51a0b5471bc813bf2)